.azure/export.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
Next Step: Wait for the export workflow run
 if the run failed, check diff file, if succeed, resolve this comment and merge the pull request